msal-dotnet-articles/acquiring-tokens/desktop-mobile/linux-broker-net.md

```diff
@@ -0,0 +1,89 @@
+---
+# Required metadata
+# For more information, see https://review.learn.microsoft.com/en-us/help/platform/learn-editor-add-metadata?branch=main
+# For valid values of ms.service, ms.prod, and ms.topic, see https://review.learn.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main
+
+title:       Use MSAL.NET on Linux with Broker
+description: "Learn how to use MSAL.NET with broker on Linux platform."
+author:      xinyuxu1026 # GitHub alias
+ms.author:   xinyuxu # Microsoft alias
+ms.service:  msal
+# ms.prod:   # To use ms.prod, uncomment it and delete ms.service
+ms.topic:    how-to
+ms.date:     05/05/2025
+---
+
+# Using MSAL .NET with an Authentication Broker on Linux
+
+
+> [!NOTE]
+> Microsoft Single Sign-on for Linux authentication broker support is introduced with `MSAL.NET` version v4.69.1.
```

Comment:
Since you're talking about a specific version, it's better to use the package name.
```suggestion
> Microsoft single sign-on for Linux authentication broker support is introduced with `Microsoft.Identity.Client` version v4.69.1.
```